src/components/builder-footer/builder-footer.js

```diff
@@ -14,7 +14,7 @@ const BuilderFooter = ({SideBarAndIsHome, HomePage}) => {
       ${SideBarAndIsHome ? builderFooterStyles.home : ''}
       `}
     >
-      <span>2019® - design by <a href="http://www.mybitdesign.com" target="_blank" rel="noreferrer noopener">www.mybitdesign.com</a></span>
+      <span>2019® - design by <a href="http://www.mybitdesign.com" target="_blank" rel="noreferrer noopener">mybitdesign</a> - built by <a href="/">datavaluepeople</a></span>
```

Comment:
I have just checked this on mobile and maybe we should add a line break on the break point sm:
https://stackoverflow.com/questions/37836821/line-break-on-mobile-phone-only

What do you think?

Reply:
![image](https://user-images.githubusercontent.com/4256007/107345517-f1a19e80-6ac3-11eb-85f3-20117f073b72.png)